main/ble2mqtt.c

```diff
@@ -847,6 +848,12 @@ void app_main()
     switch (config_network_type_get())
     {
     case NETWORK_TYPE_ETH:
+        if (config_eth_phy_power_pin_get() >= 0) {
+            gpio_pad_select_gpio(config_eth_phy_power_pin_get());
+            gpio_set_direction(config_eth_phy_power_pin_get(), GPIO_MODE_OUTPUT);
+            gpio_set_level(config_eth_phy_power_pin_get(), 1);
+            vTaskDelay(pdMS_TO_TICKS(10));
+        }
```

Comment:
Please pass `config_eth_phy_power_pin_get()` as a parameter to `eth_connect()` and set up the GPIO there